pyswmm/nodes.py

```diff
@@ -0,0 +1,626 @@
+# -*- coding: utf-8 -*-
+# -----------------------------------------------------------------------------
+# Copyright (c) 2014 Bryant E. McDonnell
+#
+# Licensed under the terms of the BSD2 License
+# See LICENSE.txt for details
+# -----------------------------------------------------------------------------
+"""Nodes module for the pythonic interface to SWMM5."""
+
+from toolkitapi import *
```

Comment:
In general using the `*` on imports is discouraged. its better to be explicit about what gets imported.